README.md

```diff
@@ -1,138 +1,70 @@
-
-<!-- GETTING STARTED
-## Getting Started
-
-this is the place to show how to use this plugin
-but i haven't tried it yet so i leave it empty lol
--->
-
-### Prerequisites
-#### Software
-* Dalamud(integrated in XIVQuickLauncher)
-  ```
-  https://github.com/goatcorp/FFXIVQuickLauncher
-  ```
-#### Plugins
-<i> You need 3 prerequisited plugins:
-1. vnavmesh
-2. BossMod or BossModReborn
-3. BossMod AutoRotation or RotationSolverReborn </i>
-
-* vnavmesh and BossMod(AutoRotation) (add repo url to dalamud)
-  ```
-  https://puni.sh/api/repository/veyn
-  ```
-
-* BossModReborn and RotationSolverReborn(add repo url to dalamud)
-  ```
-  https://raw.githubusercontent.com/FFXIV-CombatReborn/CombatRebornRepo/main/pluginmaster.json
-  ```
-
-### Installation
-
-1. add repo url to dalamud
-   ```
-   https://puni.sh/api/repository/herc
-   ```
-2. install all the prerequisited plugins mentioned in <a href="#plugins">Plugins</a>
-3. install this plugin in dalamud's plugin store
-
-<!-- CONTRIBUTING -->
-## Contributing
-
-Contributions are what make the open source community such an amazing place to learn, inspire, and create. Any contributions you make are **greatly appreciated**.
-
-If you have a suggestion that would make this better, please fork the repo and create a pull request. You can also simply open an issue with the tag "enhancement".
-Don't forget to give the project a star! Thanks again!
-
-1. Fork the Project
-2. Create your Feature Branch (`git checkout -b feature/AmazingFeature`)
-3. Commit your Changes (`git commit -m 'Add some AmazingFeature'`)
-4. Push to the Branch (`git push origin feature/AmazingFeature`)
-5. Open a Pull Request
-
-<!-- MARKDOWN LINKS & IMAGES -->
-<!-- https://www.markdownguide.org/basic-syntax/#reference-style-links -->
-[contributors-shield]: https://img.shields.io/github/contributors/ffxivcode/AutoDuty.svg?style=for-the-badge
-[contributors-url]: https://github.com/ffxivcode/AutoDuty/graphs/contributors
-[forks-shield]: https://img.shields.io/github/forks/ffxivcode/AutoDuty.svg?style=for-the-badge
-[forks-url]: https://github.com/ffxivcode/AutoDuty/network/members
-[stars-shield]: https://img.shields.io/github/stars/ffxivcode/AutoDuty.svg?style=for-the-badge
-[stars-url]: https://github.com/ffxivcode/AutoDuty/stargazers
-[issues-shield]: https://img.shields.io/github/issues/ffxivcode/AutoDuty.svg?style=for-the-badge
-[issues-url]: https://github.com/ffxivcode/AutoDuty/issues
+# AutoDuty
+
+<img align="right" width="150" height="150" src="logo.png">
+
+AutoDuty (AD) is a Dalamud plugin for FFXIV that serves as a tool to assist in the creation and following of paths through dungeons and duties. It's designed to enable automatic looping of instanced content with Duty Support, Trusts, or Squadrons.
+
+Support for it can be found in the [Puni.sh Discord server](https://discord.gg/punishxiv).
```

Comment:
these discord links i would prefer direct to channel links:
ad -> https://discord.com/channels/1001823907193552978/1236757595738476725

Reply:
Resolved in [08d171f](https://github.com/ffxivcode/AutoDuty/pull/317/commits/08d171ffdf912c94222db68a7e03508701797170)